app/routes/account.$accountId.settings.plan/components/FreePlanLimitCard/FreePlanLimitCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { Divider, Group, Text, Stack } from "@mantine/core"
+import React from "react"
+import { TitledCard } from "~/components/TitledCard"
+import { Account } from "~/models/portal/sdk"
+import { getPlanName } from "~/utils/planUtils"
+
+type FreePlanLimitCardProps = {
+  account: Account
+}
+
+export const FreePlanLimitCard = ({ account }: FreePlanLimitCardProps) => {
+  const cardItems = [
+    {
+      label: "Plan Type",
+      value: getPlanName(account.planType),
+    },
+    // {
```

Comment:
Don't we want 100,000 here?

Reply:
this is purely cosmetic -- im going to leave this out for now